### PR TITLE
Update rippled-example.cfg to clarify usage of ssl_cert and ssl_chain

### PR DESCRIPTION
**Pull request by ggprod**
## High Level Overview of Change

update notes for ssl_cert and ssl_chain to clarify that ssl_chain must be used when the certificate includes one or more intermediates

### Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactor (non-breaking change that only restructures code)
- [ ] Tests (You added tests for code that already exists, or your new feature included in this PR)
- [x] Documentation Updates
- [ ] Release

### CODE REVIEW

File: cfg/rippled-example.cfg [hunk 1]
```diff
@@ -283,12 +283,14 @@
 #       ssl_cert
 #
 #           Specifies the path to the SSL certificate file in PEM format.
-#           This is not needed if the chain includes it.
+#           This is not needed if the chain includes it.  Use ssl_chain if
+#           your certificate includes one ore more intermediates
```
Comment (intelliot) [inline cfg/rippled-example.cfg:287]:
```suggestion
#           your certificate includes one ore more intermediates.
```

File: cfg/rippled-example.cfg [hunk 2]
```diff
@@ -283,12 +283,14 @@
...
 #
 #       ssl_chain
 #
 #           If you need a certificate chain, specify the path to the
 #           certificate chain here. The chain may include the end certificate.
+#           This must be used if the certificate includes intermediates
```
Comment (intelliot) [inline cfg/rippled-example.cfg:293]:
```suggestion
#           This must be used if the certificate includes intermediates.
```
Most sentences in this file are ended with a period (`.`) so I figure we should remain consistent.

**Comment by ggprod** [comment]
@intelliot should I add another comment on ssl_verify (when set to 1 it seems to prevent commands like `rippled server_info` from working)?

**Comment by ckeshava** [comment]
@ggprod I have observed this error when the local rippled server was stopped (Through SIGKILL or the `rippled stop` command)
When you use curl, you are connecting to a non-local rippled server right?

**Review by ckeshava — APPROVED** [review]

**Comment by ckeshava** [comment]
I'm surprised to see these two errors in the CI tests, despite not having any code changes. Do you observe this in your local unit tests too? @ggprod 
```
1> #8 failed: ShardArchiveHandler_test.cpp(420)
1> #9 failed: ShardArchiveHandler_test.cpp(427)
```

**Comment by ggprod** [comment]
> I'm surprised to see these two errors in the CI tests, despite not having any code changes. Do you observe this in your local unit tests too? @ggprod
> 
> ```
> 1> #8 failed: ShardArchiveHandler_test.cpp(420)
> 1> #9 failed: ShardArchiveHandler_test.cpp(427)
> ```

I didn't run unit tests as I only made changes in that one file (and only among the comments)

**Comment by ggprod** [comment]
> @ggprod I have observed this error when the local rippled server was stopped (Through SIGKILL or the `rippled stop` command) When you use curl, you are connecting to a non-local rippled server right?

in my case rippled was running fine and was responding to the same request when done using curl (either internally via localhost or externally)

**Comment by ggprod** [comment]
I am closing this PR and will re-do it as I didn't sign my commit

**Comment by intelliot** [comment]
Replaced by 4667

**Comment by intelliot** [comment]
Replaced by #4667

**Pull request closed without merging**